symfony/asset-mapper/6.4/manifest.json

```diff
@@ -16,7 +16,7 @@
         {
             "file": "templates/base.html.twig",
             "content": "{% block importmap %}{{ importmap('app') }}{% endblock %}",
-            "position": "after_target",
+            "position": "before_target",
```

Comment:
Interesting situation :). But (see error above), there is no `before_target` - there just wasn't a need for it. Either we need to think of some other solution or add that to Flex.